.travis.yml

```diff
@@ -1,6 +1,15 @@
 sudo: false
 language: java
+
+# We use OpenJDK instead of Oracle JDK because ya all should only
+# ever use OpenJDK!  (And get Red Hat's support for it, if
```

Comment:
Is it really necessary ??

Reply:
@sylvainlaurent maybe not... ;-) removed! Merge? (Please Squash.)